analytics-integrations/kahuna/src/main/java/com/segment/analytics/internal/integrations/KahunaIntegration.java

```diff
@@ -83,4 +149,14 @@
 
     KahunaAnalytics.logout();
   }
+
+  @Override public void alias(AliasPayload alias) {
+    KahunaAnalytics.setUserCredential(USER_ID_KEY, alias.userId());
```

Comment:
This doesn't look right - if you don't support alias don't implement this method. Users are instructed to call `identify` after calling alias so you'll get the new user id anyway. `Alias` is reserved when you explicitly want to link two IDs, previousId and currentId together.